### PR TITLE
Update ORT to 1.12 release

### PR DESCRIPTION
**Pull request by pranavsharma**
ORT 1.12 release branch is ready for validation. We have not released yet. Release is scheduled for 7/21.

### CODE REVIEW

File: build.py [hunk 1]
```diff
@@ -95,7 +95,7 @@
     '2.24.0dev': (
         '22.07dev',  # triton container
         '22.06',  # upstream container
-        '1.11.1',  # ORT
+        '1.12',  # ORT
```
Comment (tanmayv25) [inline build.py:98]: 1.12.0 ?
https://github.com/microsoft/onnxruntime/tree/rel-1.12.0

**Comment by tanmayv25** [comment]
Curious, can we advance the ORT Openvino version with ORT 1.12.0? I remember that the ORT 2022.1 was supported only on intel's flavor of ORT 1.11.1

**Comment by pranavsharma** [comment]
> Curious, can we advance the ORT Openvino version with ORT 1.12.0? I remember that the ORT 2022.1 was supported only on intel's flavor of ORT 1.11.1

Looks like we can. I'll update the PR.

File: build.py [hunk 2]
```diff
@@ -95,8 +95,8 @@
     '2.24.0dev': (
         '22.07dev',  # triton container
         '22.06',  # upstream container
-        '1.11.1',  # ORT
-        '2021.4.582',  # ORT OpenVINO
+        '1.12.0',  # ORT
+        '2022.1.0',  # ORT OpenVINO
         (('2021.4', None), ('2021.4', '2021.4.582'),
```
[the remark this answers is not in the document]
Comment (pranavsharma) [reply build.py:100]: @tanmayv25 ?

[the remark this answers is not in the document]
Comment (tanmayv25) [reply build.py:100]: Those are for our standalone openvino backend versions.
https://github.com/triton-inference-server/openvino_backend
@pranavsharma  Can you revert the ORT OpenVINO update? We would need to do this update on Triton side. 
Please take a look at the description here: https://github.com/triton-inference-server/server/blob/main/build.py#L67-L70

[the remark this answers is not in the document]
Comment (tanmayv25) [reply build.py:100]: Did we have a location we can track support of openVINO in generic ORT release?

[the remark this answers is not in the document]
Comment (pranavsharma) [reply build.py:100]: The requirements should be listed here https://onnxruntime.ai/docs/execution-providers/OpenVINO-ExecutionProvider.html#requirements

**Comment by pranavsharma** [comment]
ORT 1.12 release branch is finalized. Are we going to pick up this PR for the July Triton release?

**Comment by tanmayv25** [comment]
@pranavsharma Yes. We already have picked these changes in our 22.07 branches.
https://github.com/triton-inference-server/server/commit/a00aa44426c68f4d2ebd253b0abcefec6ac87857

You can close this PR. We will advance our main branch to ORT 1.12.0 sometime later.

**Comment by tanmayv25** [comment]
@pranavsharma Closing the PR as the ORT version was updated using this commit:  https://github.com/triton-inference-server/server/commit/b498e31d27eab444fc7725ab201bc162ce4a404d

**Pull request closed without merging**